include/cbl/CBLQuery.h

```diff
@@ -41,8 +41,8 @@ CBL_CAPI_BEGIN
 
 /** Query languages */
 typedef CBL_ENUM(uint32_t, CBLQueryLanguage) {
-    kCBLJSONLanguage,   ///< [JSON query schema](https://github.com/couchbase/couchbase-lite-core/wiki/JSON-Query-Schema)
-    kCBLN1QLLanguage    ///< [N1QL syntax](https://docs.couchbase.com/server/6.0/n1ql/n1ql-language-reference/index.html)
+    kCBLJSONLanguage,       ///< [JSON query schema](https://github.com/couchbase/couchbase-lite-core/wiki/JSON-Query-Schema)
+    kCBLStringLanguage      ///< [N1QL syntax](https://docs.couchbase.com/server/6.0/n1ql/n1ql-language-reference/index.html)
```

Comment:
The name is little odd to me. It makes me wonder if kCBLJSONLanguage is a graphical language. If avoiding branding name (n1ql or sql), how about CBLSequelLanguage?

Reply:
I think SequelLanguage is the same as SQL when pronouncing it?

Reply:
Yes, but it avoids the connotation of proper noun of SQL

Reply:
Why would we not use "N1QL" in the API?! It _is_ N1QL, with a few differences. (And Server already has at least two flavors of N1QL that aren't identical.) I can't think of anything else we could call it that would make sense.

Reply:
I will revert the change back to use N1QL. I personally agreed with you, and I have just started a separate discussion to insist to use N1QL as a name in the language options here. Once that is finalized and if the change needs to be made, the separate PR will be created.

Reply:
OK, I reverted back to use N1QL until it's really finalized.